src/Polytopes/Cone.jl

```diff
@@ -0,0 +1,112 @@
+@doc Markdown.doc"""
+    Cone(Rays)
+
+     A polyhedral cone, not necessarily pointed, defined by the positive hull
+     of the rays, Rays.
+"""
+struct Cone #a real polymake polyhedron
+    pm_cone::Polymake.BigObjectAllocated
+end
+function Cone(Rays::Union{Oscar.MatElem,AbstractMatrix})
+    Cone(Polymake.polytope.Cone{Polymake.Rational}(
+        RAYS = matrix_for_polymake(Rays),
+    ))
+end
+
+"""
+    pm_polytope(C::Cone)
+
+Get the underlying polymake `Cone`.
+"""
+pm_cone(C::Cone) = C.pm_cone
+
+struct ConeRayIterator
+    cone::Cone
+end
+
+function Base.iterate(iter::ConeRayIterator, index = 1)
+    rays = pm_cone(iter.cone).RAYS
+    if size(rays, 1) < index
+        return nothing
+    end
+
+    return (rays[index, :], index + 1)
```

Comment:
Seems this iterator is untested (according to the codecov report). Perhaps a test could be added (doesn't have to be in this PR)